design/0000-launcher.md

```diff
@@ -0,0 +1,120 @@
+
+The Go Initialization Layer is ideally used with environment variables; as such, not all configuration options may be available to set directly within code. If there is more desire to use configuration options directly within code, either the regular SDK can be used, or a vendor package can be used that allows for extra configuration options.
+
+## Motivation
+
+The goal of this Initialization Layer is to help users that aren't familiar with OpenTelemetry quickly ramp up on what they need to get going and instrument their applications. [There is currently a lot of boilerplate code required to use OpenTelemetry](https://opentelemetry.io/docs/instrumentation/go/manual/#initializing-a-new-tracer), and users must make decisions early on before instrumentation works - for example, they must determine and find the appropriate exporters, set resource attributes, etc. This Initialization Layer is intended to simplify the setup and minimize decision-making overhead to be able to quickly get started.
+
+Separately, vendors may choose to provide a vendor-specific package compatible with this Initialization Layer for easier exporting to their own backend. Neither the Initialization Layer nor the vendor-specific package are required to use OpenTelemetry, and using the Initialization Layer does not require the end user to use a vendor-specific package.
+
+## Explanation
+
+The Go Initialization Layer provides a single function to reduce the amount of code that needs to be written to get started, and provides additional, straightforward options to add in that still make additional configuration easy.
+
+The vendor-neutral Initialization Layer will live in the opentelemetry-go-contrib repo, and vendors may create separately hosted vendor-specific configuration packages that can be used with the Initialization Layer.
+
+### Getting started
+
+```shell
+go get go.opentelemetry.io/contrib/otelinit
+```
+
+### Configure
+
+Minimal setup - by default will send all telemetry to `localhost:4317`, with the ability to set environment variables for endpoint, headers, resource attributes, and more as listed in the configuration options noted below:
+
+```go
+import "go.opentelemetry.io/contrib/otelinit"
+
+func main() {
+    init, err := otelinit.ConfigureOpentelemetry()
+    defer init.Shutdown()
+}
+```
+
+Alternatively, set environment variables or set variables directly in code to override defaults:
+
+```go
+import "go.opentelemetry.io/contrib/otelinit"
+
+func main() {
+    init, err := otelinit.ConfigureOpentelemetry(
+        otelinit.WithTracesExporter(new trace.SpanExporter("jaeger")) // if using non-default exporter
+        otelinit.WithPropagators(b3.New()) // if using non-default propagator
```

Comment:
do the imports for `trace` and `b3` come from api / sdk or something under `otelinit`?